packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +166,18 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		workflowRunning: {
```

Comment:
this should not work if node is disabled

Reply:
or if there's multiple trigger nodes (based on notion page)

Reply:
The watcher is fixed. Now it shows the tooltip in this cases:

#workflow is runnung (bool)
#if it is a trigger node (this.isWebhookNode)
#if it is only one trigger node in the WF (this.isSingleWebhookNode)
#if the node is not disabled (!this.data.disabled)

Reply:
there's still one scenario here... if you have one disabled and one not disabled node, then it should still show tooltip above the active trigger.

Reply:
After re-work and moving some of the logic in the $store, this is fixed.